hack/determine_install_upgrade_version.py

```diff
@@ -0,0 +1,55 @@
+import yum
+import sys
+import os.path
+from version_utils import rpm
```

Comment:
They are... at least I was using them and needed to install the `version_utils` via pip